templates/zerver/integrations/notion.md

```diff
@@ -0,0 +1,32 @@
+Get Zulip notifications for your Notion database via Zapier!
+
+1. {!create-stream.md!}
+
+1. {!create-an-incoming-webhook.md!}
+
+1. Create an account on [Zapier](https://zapier.com).
+
+1. [Click here](https://zapier.com/developer/public-invite/8304/bb0e9784d171eb44762c1bef4fcba2df/)
+   and then click **Accept Invite & Build a Zap**.
+
+1. Click **Create Zap** in the upper left.
+
+1. Now select **Notion** as the trigger app and choose an event. For
+   example, you can choose **New Database Item** as the event.
+
+1. After this, you will be asked to connect your **Notion** account.
+   Follow the instructions to do so.
+
+1. After connecting your account, you will be asked to select a
+   database for which you want to get notifications.
+   Now, you will be asked to test the trigger. If the test is
```

Comment:
Maybe: 
Now, you will be asked to test the trigger. -> You will then be asked to test the trigger.